app/static/js/int/main.js

```diff
@@ -0,0 +1,171 @@
+/*
+
+>> kasperkamperman.com - 2018-04-18
+>> https://www.kasperkamperman.com/blog/camera-template/
+
+*/
```

Comment:
What is this for? Is the code copyrighted? If so, we shouldn't just be pasting it into our code.